src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -234,66 +240,107 @@ export class StackDetailsComponent implements OnInit {
     this.currentStackRows = [];
     for (let component in components) {
       if (components.hasOwnProperty(component)) {
-        this.currentStackRows.push({ name: components[component].name, version: components[component].version });
+        this.currentStackRows.push({ name: components[component].name,
+          version: components[component].version });
       }
     }
   }
 
+  // TODO: To be removed after the demo
+  private fetchStaticRecommendation(): any {
+    return {
+            'recommendations': {
+                'similar_stacks': [
+                    {
+                        'analysis': {
+                            'version_mismatch': {
+                                'vertx:vertx-web-templ-freemarker': '3.3.4',
+                                'vertx:vertx-web-templ-mvel': '3.4.0'
+                            },
+                            'missing_packages': {
+                                'vertx:vertx-mongo-embedded-db' :'3.3.3'
```

Comment:
@joshuawilson This is eventually going to be removed as it is just a temporary thing.